tf2_ros/include/tf2_ros/transform_listener.h

```diff
@@ -116,6 +116,7 @@ class TransformListener
   rclcpp::Subscription<tf2_msgs::msg::TFMessage>::SharedPtr message_subscription_tf_static_;
   tf2::BufferCore & buffer_;
   bool using_dedicated_thread_ = false;
+  std::atomic<bool> stop_thread_{false};
```

Comment:
great!
For consistency, do you mind changing the previous `bool using_dedicated_thread_` to be initialized consistently?

Reply:
Done